lib/modules/platform/pr-body.spec.ts

```diff
@@ -98,4 +128,75 @@ describe('modules/platform/pr-body', () => {
       });
     });
   });
+
+  describe('updateRenovateBody', () => {
+    test.each([
```

Comment:
use table string template syntax, which is more readable

Reply:
f92b7cbe3dbd5e1ad367de65de5874affc02c644